themes/build.js

```diff
@@ -0,0 +1,96 @@
+const request = require("request");
+const less = require("less");
+const fs = require("fs");
+const { variants, labels } = require("@catppuccin/palette");
+
+const themes = ["latte", "frappe", "macchiato", "mocha"];
+
+const url = "https://raw.githubusercontent.com/catppuccin/palette/main/less/";
+
+fs.mkdirSync("built");
+
+console.log("Reading theme.less file");
+fs.readFile("src/theme.less", "utf8", (err, data) => {
+	if (err) {
+		console.error(err);
+		return;
+	}
+	// iterate through themes
+	themes.forEach((theme) => {
```

Comment:
This is pretty messy. Perhaps separate components into functions?

Reply:
Would something like this make it less one after another?
```js
function handleTheme(theme) {
    ...
}

themes.forEach(handleTheme);
```

Reply:
Done! With great advice from @andreasgrafen, I removed all requests and divided the program into respective functions. Lmk if this isn't what you had in mind